app/models/declaration_state.rb

```diff
@@ -3,18 +3,21 @@
 class DeclarationState < ApplicationRecord
   belongs_to :participant_declaration
 
+  include PGEnum(state_reason: %w[duplicate])
```

Comment:
Do we want to expose this in the serializer yet, or are we going to hold off for now?

Reply:
We've agreed with Nathan to hold off for now, until a provider actually ask for something.